timemachine/fe/system.py

```diff
@@ -0,0 +1,61 @@
+import functools
+
+import jax.numpy as jnp
+import numpy as np
+
+from timemachine.potentials import bonded, nonbonded
+
+
+class VacuumSystem:
+
+    # utility system container
+
+    def __init__(self, bond, angle, torsion, nonbonded):
+        self.bond = bond
+        self.angle = angle
+        self.torsion = torsion
+        self.nonbonded = nonbonded
+
+    def get_U_fn(self):
```

Comment:
(deboggle): Seems to overlap with enhanced.VacuumState: https://github.com/proteneer/timemachine/blob/master/timemachine/md/enhanced.py#L63

Reply:
ugh - good call. We should deboggle this for sure - but I didn't want to touch production code for this PR yet.